.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "Python 3",
+	"image": "mcr.microsoft.com/devcontainers/python:1-3.11-bullseye",
+	"postCreateCommand": "pip3 install --user .[dev]",
```

Comment:
We need the ["test"] dependencies as well. How should this be organized in a proper way?

Reply:
![image](https://github.com/DHI/modelskill/assets/614215/258550a6-96ea-4e1b-9fe4-19b26d5e0a31)

Reply:
Oh, yeah, we need to be able to run those tests 👍

Reply:
I see 👍 Shouldn't the dev requirements then maybe contain all the test requirements as well? I guess any developer wants to be able to run the tests locally also

Reply:
I guess so but does .[dev,test] not work?